src/config/resolve-config.js

```diff
@@ -126,13 +125,11 @@ resolveConfigFile.sync = filePath => {
 };
 
 function mergeOverrides(configResult, filePath) {
-  const options = Object.assign({}, configResult.config);
-  if (filePath && options.overrides) {
-    const relativeFilePath = path.relative(
-      path.dirname(configResult.filepath),
-      filePath
-    );
-    for (const override of options.overrides) {
+  const { config } = configResult || {};
+  const { overrides, filepath: configPath, ...options } = config || {};
```

Comment:
It was `configResult.filepath`, not `configResult.config.filepath`.

Reply:
Fixed, thanks.